README.md

```diff
@@ -1,6 +1,7 @@
 # implicit-hie
 ```bash
 cd your-stack-or-cabal-package
+stack install implicit-hie OR cabal install implicit-hie
```

Comment:
```suggestion
stack install implicit-hie # or cabal install implicit-hie
```
Being a bash code snippet it is usual try to make it runnable without errors even if you copy the entire line